player/js/module_worker.js

```diff
@@ -182,20 +182,22 @@ function workerContent() {
       animation.onError = function (error) {
         console.log('ERRORO', error); // eslint-disable-line
       };
-      if (params.renderer === 'svg') {
-        animation.addEventListener('DOMLoaded', function () {
+      animation.addEventListener('DOMLoaded', function () {
+        if (params.renderer === 'svg') {   
           var serialized = wrapper.serialize();
           addElementToList(wrapper, elements);
-          self.postMessage({
-            type: 'loaded',
-            payload: {
-              id: payload.id,
-              tree: serialized.children[0],
-              totalFrames: animation.totalFrames,
-              frameRate: animation.frameRate,
-            },
-          });
+        }
+        self.postMessage({
+          type: 'loaded',
+          payload: {
+            id: payload.id,
+            tree: serialized?.children[0],
```

Comment:
Not sure how much transpilation is happening here, let's do

```suggestion
            tree: serialized && serialized.children[0],
```